lib/Conversion/PipelineToHW/PipelineToHW.cpp

```diff
@@ -182,20 +251,20 @@ class PipelineLowering {
 
   // Name of this pipeline - used for naming stages and registers.
   // Implementation defined.
-  std::string pipelineName;
+  StringAttr pipelineName;
 };
 
 class PipelineInlineLowering : public PipelineLowering {
 public:
   using PipelineLowering::PipelineLowering;
 
   StringAttr getStageRegPrefix(size_t stageIdx) override {
-    return builder.getStringAttr(pipelineName + "_s" +
-                                 std::to_string(stageIdx));
+    return builder.getStringAttr(pipelineName.strref() + "_s" +
```

Comment:
I'm surprised you need the strref()

Reply:
Yerp, no `Twine` constructor in `StringAttr`.